src/index.ts

```diff
@@ -0,0 +1,99 @@
+import { getDependencies } from './getdependencies';
+import { getRepoIds } from './getrepos';
+const JupiterOneClient = require('@jupiterone/jupiterone-client-nodejs');
+const gql = require("graphql-tag");
+
+const J1_USER_POOL_ID_PROD = "us-east-2_9fnMVHuxD";
+const J1_CLIENT_ID_PROD = "1hcv141pqth5f49df7o28ngq1u";
+const accessToken = '';
+
+let success = 0, failure = 0;
+const missingDeps = [];
+
+;(async () => {
+  const j1Client = await new JupiterOneClient({
+    account: 'j1dev',
+    username: 'j1dev',
+    password: 'dev',
+    poolId: J1_USER_POOL_ID_PROD,
+    clientId: J1_CLIENT_ID_PROD,
+    accessToken: accessToken,
```

Comment:
I may be missing something but how do you intend for this to be set?

Reply:
Also, you can say just `accessToken` instead of `accessToken: accessToken`.

Reply:
I've made it so that it use env to find poolId, clientId, and accessToken

Reply:
I see where it uses env vars to find poolId and clientId, but where are you setting the accessToken variable to a value from the env? Maybe I'm dumb and missed it. Could you give a me a link to the line where that happens? :sweat_smile:

Reply:
Oh yeah I'm working on that right now it wasn't in this commit